templates/js.html

```diff
@@ -2,13 +2,15 @@
 <html>
 <head>
     <script src="/templates/logger.js"></script>
-<script src="/templates/app.js"></script>
+    <script src="/templates/app.js"></script>
 </head>
 <body>
 This is a function file
 
-<script>
+<script type="module">
     {{EMBEDDED_CODE}}
+
+    window.main = main;
```

Comment:
The non-implicit `window` in module script tags definitely through me off initially